ParcelDistributionCenter/ParcelDistributionCenter.Logic/PackageHandler.cs

```diff
@@ -20,6 +22,8 @@ public static Package FindPackageByNumber(int packageNumber)
 
         public static IEnumerable<Package> FindPackagesBySenderEmail(string senderEmail) => ReturnPackages(p => p.SenderEmail == senderEmail);
 
+        public static IEnumerable<Package> FindAll() =>ReturnPackages(p => true);
+
```

Comment:
To myślę mogłoby być po prostu FindAll() => MemoryRepository.PackagesList;

Reply:
Ok. Do zmiany.